valhalla/baldr/rapidjson_utils.h

```diff
@@ -28,9 +28,12 @@
 
 namespace rapidjson {
 
-template <typename T> inline std::string to_string(const T& document_or_value) {
+template <typename T>
+inline std::string to_string(const T& document_or_value, int decimal_places = -1) {
   rapidjson::StringBuffer buffer;
   rapidjson::Writer<rapidjson::StringBuffer> writer(buffer);
+  if (decimal_places >= 0)
+    writer.SetMaxDecimalPlaces(decimal_places);
```

Comment:
this makes it easy for us to truncate all precision to a certain value, which is nice for geojson where we can just say we only care about 5 places for example